introduction.md

```diff
@@ -1,4 +1,8 @@
 That's me
 
 My favorite Code School path is Ruby. Or is it Electives? Or HTML & CCC?
-I don't know.
\ No newline at end of file
+I don't know.
+
+My deadly skills:
+* A bit of Ruby
+* Some Git
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?